src/routes.js

```diff
@@ -17,7 +17,7 @@ the application has.
 export const Router = () => {
   return (
     <Routes>
-      <Route exact path="/" element={routes.about} />
+      <Route exact path="/" element={routes[0].about} />
       {Object.keys(routes).map((route, index) => (
```

Comment:
Amg essa parte das routes seria melhor se fosse feito dessa forma:

```suggestion
      {routes.map((route, index) => (
         <Route key={index} path={route.path} element={route.component} />
      )
```